EIPS/eip-721.md

```diff
@@ -28,17 +28,17 @@ In general, all houses are distinct and no two kittens are alike. NFTs are *dist
 
 ## Motivation
 
-A standard interface allows wallet/broker/auction applications to work with any NFT on Ethereum. We provide for simple [ERC-721](eip-721) smart contracts as well as contracts that track an *arbitrarily large* number of NFTs. Additional applications are discussed below.
+A standard interface allows wallet/broker/auction applications to work with any NFT on Ethereum. We provide for simple non-fungible token smart contracts as well as contracts that track an *arbitrarily large* number of NFTs. Additional applications are discussed below.
 
-This standard is inspired by the [ERC-20](eip-20) token standard and builds on two years of experience since EIP-20 was created. EIP-20 is insufficient for tracking NFTs because each asset is distinct (non-fungible) whereas each of a quantity of tokens is identical (fungible).
+This standard is inspired by the [ERC-20](./eip-20) token standard and builds on two years of experience since EIP-20 was created. EIP-20 is insufficient for tracking NFTs because each asset is distinct (non-fungible) whereas each of a quantity of tokens is identical (fungible).
```

Comment:
```suggestion
This standard is inspired by the [ERC-20](./eip-20.md) token standard and builds on two years of experience since EIP-20 was created. EIP-20 is insufficient for tracking NFTs because each asset is distinct (non-fungible) whereas each of a quantity of tokens is identical (fungible).
```